src/nvim/os/stdpaths.c

```diff
@@ -8,28 +8,35 @@
 
 /// Names of the environment variables, mapped to XDGVarType values
 static const char *xdg_env_vars[] = {
+#ifdef WIN32
+  [kXDGConfigHome] = "LOCALAPPDATA",
+  [kXDGDataHome] = "LOCALAPPDATA",
+  [kXDGCacheHome] = "TEMP",
+  [kXDGRuntimeDir] = NULL,
+  [kXDGConfigDirs] = NULL,
+  [kXDGDataDirs] = NULL,
```

Comment:
Do we need another fallback on `~\\Appdata\\Local`, or can the Windows fallback just be `os_getenv('$LOCALAPPDATA')`?